docs/ui/installation.rst

```diff
@@ -31,8 +31,22 @@ Once the server is running, you can navigate to:
 Configuration
 =============
 
-The sample configuration makes the UI load job/push data from the production service API. If you wish to test the UI against stage/dev's service instead, adjust ``thServiceDomain`` in the config file created as part of installation:
-``ui/js/config/local.conf.js``
+The sample configuration makes the UI load job/push data from the production service API.
```

Comment:
We should probably say something like this instead (since we're no longer using a sample configuration):

```
By default, the web server will serve data from treeherder production. If you wish to test the UI against treeherder stage instead, try:
```